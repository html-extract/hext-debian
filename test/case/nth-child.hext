<ul class:div>
  <li:nth-child(3n+1) class:link/>
</ul>

