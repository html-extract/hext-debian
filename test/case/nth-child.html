<html>
  <head>
    <style type="text/css">
      ul {
        list-style:numeric;
      }
      li:after {
        content: "node";
      }
      li:nth-child(-2n+5) {
        color:red;
      }
    </style>
  </head>
  <body>
    <ul class="anchor1 first">
      <li class="child1 first"></li>
      <li class="child2"></li>
      <li class="child3"></li>
      <li class="child4"></li>
      <li class="child5"></li>
      <li class="child6"></li>
      <li class="child7"></li>
      <li class="child8"></li>
      <li class="child9"></li>
      <li class="child10"></li>
      <li class="child11"></li>
      <li class="child12"></li>
      <li class="child13"></li>
      <li class="child14 last"></li>
    </ul>
    <ul class="anchor2">
      <li class="child1 first"></li>
      <li class="child2"></li>
      <li class="child3"></li>
      <li class="child4"></li>
      <li class="child5"></li>
      <li class="child6"></li>
      <li class="child7"></li>
      <li class="child8"></li>
      <li class="child9"></li>
      <li class="child10"></li>
      <li class="child11"></li>
      <li class="child12"></li>
      <li class="child13"></li>
      <li class="child14 last"></li>
    </ul>
    <ul class="anchor3">
      <li class="child1 first"></li>
      <li class="child2"></li>
      <li class="child3"></li>
      <li class="child4"></li>
      <li class="child5"></li>
      <li class="child6"></li>
      <li class="child7"></li>
      <li class="child8"></li>
      <li class="child9"></li>
      <li class="child10"></li>
      <li class="child11"></li>
      <li class="child12"></li>
      <li class="child13"></li>
      <li class="child14 last"></li>
    </ul>
    <ul class="anchor4">
      <li class="child1 first"></li>
      <li class="child2"></li>
      <li class="child3"></li>
      <li class="child4"></li>
      <li class="child5"></li>
      <li class="child6"></li>
      <li class="child7"></li>
      <li class="child8"></li>
      <li class="child9"></li>
      <li class="child10"></li>
      <li class="child11"></li>
      <li class="child12"></li>
      <li class="child13"></li>
      <li class="child14 last"></li>
    </ul>
    <ul class="anchor5">
      <li class="child1 first"></li>
      <li class="child2"></li>
      <li class="child3"></li>
      <li class="child4"></li>
      <li class="child5"></li>
      <li class="child6"></li>
      <li class="child7"></li>
      <li class="child8"></li>
      <li class="child9"></li>
      <li class="child10"></li>
      <li class="child11"></li>
      <li class="child12"></li>
      <li class="child13"></li>
      <li class="child14 last"></li>
    </ul>
    <ul class="anchor6">
      <li class="child1 first"></li>
      <li class="child2"></li>
      <li class="child3"></li>
      <li class="child4"></li>
      <li class="child5"></li>
      <li class="child6"></li>
      <li class="child7"></li>
      <li class="child8"></li>
      <li class="child9"></li>
      <li class="child10"></li>
      <li class="child11"></li>
      <li class="child12"></li>
      <li class="child13"></li>
      <li class="child14 last"></li>
    </ul>
    <ul class="anchor7 last">
      <li class="child1 first"></li>
      <li class="child2"></li>
      <li class="child3"></li>
      <li class="child4"></li>
      <li class="child5"></li>
      <li class="child6"></li>
      <li class="child7"></li>
      <li class="child8"></li>
      <li class="child9"></li>
      <li class="child10"></li>
      <li class="child11"></li>
      <li class="child12"></li>
      <li class="child13"></li>
      <li class="child14 last"></li>
    </ul>
  </body>
</html>
