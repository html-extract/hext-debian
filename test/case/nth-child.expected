{"div":"anchor1 first","link":["child1 first","child4","child7","child10","child13"]}
{"div":"anchor2","link":["child1 first","child4","child7","child10","child13"]}
{"div":"anchor3","link":["child1 first","child4","child7","child10","child13"]}
{"div":"anchor4","link":["child1 first","child4","child7","child10","child13"]}
{"div":"anchor5","link":["child1 first","child4","child7","child10","child13"]}
{"div":"anchor6","link":["child1 first","child4","child7","child10","child13"]}
{"div":"anchor7 last","link":["child1 first","child4","child7","child10","child13"]}
